app/model/des/DirectDebitData.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model.des
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class DirectDebitData(directDebitDetails: Option[List[DirectDebitDetails]]) {
```

Comment:
Maybe we can redefine it as `List[...]` only? `Some(Nil)` is pretty much the same as `None` here.